wiki/Contests/en.md

```diff
@@ -2,291 +2,420 @@
-| 8 | [raja_ffm - the light](https://osu.ppy.sh/beatmapsets/299224) | ![osu!][osu!] | [Damnae](https://osu.ppy.sh/users/989377) | [Results](https://osu.ppy.sh/home/news/2015-04-08-monthly-beatmapping-contest-8-results) | [Score #8](https://osu.ppy.sh/p/contestresults?c=17) |
-| 9 | [Furries in a Blender - Storm World](https://osu.ppy.sh/beatmapsets/319473) | ![osu!taiko][osu!taiko] | [Firce777](https://osu.ppy.sh/users/274072) | [Results](https://osu.ppy.sh/home/news/2015-06-27-monthly-beatmapping-contest-9-results) | [Score #9](https://osu.ppy.sh/p/contestresults?c=18) |
-| 10 | [Hylian Lemon - Foresight Is for Losers](https://osu.ppy.sh/beatmapsets/342751) | ![osu!catch][osu!catch] | [ZiRoX](https://osu.ppy.sh/users/200768) | [Results](https://osu.ppy.sh/home/news/2015-08-06-monthly-beatmap-contest-10-results-ctb) | [Score #10](https://osu.ppy.sh/p/contestresults?c=19) |
-| 11 | [Freedom Planet - Dragon Valley (Toni Leys Remix feat. Esteban Bellucci)](https://osu.ppy.sh/beatmapsets/385056) | ![osu!mania][osu!mania] | [LordRaika](https://osu.ppy.sh/users/3265023) | [Results](https://osu.ppy.sh/home/news/2015-12-03-monthly-beatmapping-contest-11-results) | [Score #11](https://osu.ppy.sh/p/contestresults?c=20) |
-| 12 | [IAHN - Candy Luv](https://osu.ppy.sh/beatmapsets/456054) | ![osu!][osu!] | [Taeyang](https://osu.ppy.sh/users/2732340) | [Results](https://osu.ppy.sh/home/news/2016-05-26-monthly-beatmapping-contest-12-results) | [Score #12](https://osu.ppy.sh/p/contestresults?c=21) |
-
-Notes:
-
-- Winners of the MBC #1 to #13 received the *Elite Mapper* forum title.
-- Winning beatmaps from MBC #1 to #12 (excluding MBC #6) were bundled with the [osu!installer](https://osu.ppy.sh/home/download). They have since been replaced by other beatmaps.
-- Monthly Beatmapping Contests after #13 have a different premise and reward scheme from the previous contests detailed in the [Monthly Beatmapping Contest](Monthly_Beatmapping_Contest) article.
-
-## Aspire
-
-Aspire is a beatmapping contest with one special exception, they are not bound to the [Ranking Criteria](/wiki/Ranking_Criteria).
-
-| Contest No. | Beatmap | Mode | Mapper | Results | Scores |
-| :-: | :-- | :-: | :-- | :-- | :-- |
-| Aspire 2016 | [IAHN - Transform (Original Mix)](https://osu.ppy.sh/beatmapsets/484689) | ![osu!][osu!] | [Monstrata](https://osu.ppy.sh/users/2706438) | [Results](https://osu.ppy.sh/home/news/2016-07-23-aspire-2-concludes-winners-announced) | [Score #1](https://osu.ppy.sh/p/contestresults?c=22) |
-
-Notes:
+Official contests are contests that are being hosted (or otherwise recognized as official) by the [osu!team](wiki/People/The_Team). Most official contests have their own listing in the [contests page](https://osu.ppy.sh/community/contests) dedicated to them.
```

Comment:
```suggestion
Official contests are contests that are hosted (or otherwise recognised as official) by the [osu! team](wiki/People/The_Team). Most official contests have their own listing in the [contests page](https://osu.ppy.sh/community/contests) dedicated to them.
```